checkstyle.xml

```diff
@@ -180,15 +190,6 @@
             <property name="tagOrder" value="@param, @return, @throws, @deprecated"/>
             <property name="target" value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF, VARIABLE_DEF"/>
         </module>
-        <module name="JavadocMethod">
```

Comment:
So, I couldn't get this configuration to work in Quilt. Instead, I had to introduce the following:

```java
  <module name="JavadocMethod">
            <property name="scope" value="public"/>
            <property name="allowMissingParamTags" value="true"/>
            <property name="allowMissingReturnTag" value="true"/>
            <property name="allowedAnnotations" value="Override, Test"/>
            <property name="validateThrows" value="false"/>
        </module>
        <module name="MissingJavadocMethodCheck">
            <property name="scope" value="public"/>
            <property name="minLineCount" value="2"/>
            <property name="allowMissingPropertyJavadoc" value="true"/>
            <property name="allowedAnnotations" value="Override, Test"/>
        </module>
```

Reply:
Thank you. I thought I had figured this out but didn't quite make it there. 

Cheers for doing the legwork.